src/components/Molecule/Task/index.jsx

```diff
@@ -0,0 +1,65 @@
+import React, { useRef, useEffect, useState } from "react";
+import styled from "styled-components";
+import StyledCheckBoxWrapper from "../../Atoms/CheckBox/index";
+import Input from "../../Atoms/Input/index.jsx";
+import StyledEditButtonWrapper from "../../Atoms/EditButton/index";
+import COLOR from "../../../variables/color";
+import TEXT from "../../../variables/texts";
+
+const TaskSample = ({ defaultValue, defaultIsEditing, OnTaskChange }) => {
+  const [isEditing, StateisEditing] = useState(defaultIsEditing);
+
+  const OnTaskComplete = () => {
+    StateisEditing(false);
+    OnTaskComplete();
+  };
+
+  const OnEditButtonClick = () => {
+    StateisEditing(true);
+  };
+
+  const OnEditComplete = (value) => {
+    StateisEditing(false);
+    OnTaskChange(value);
+  };
+
+  return (
+    <StyledWrapper>
+      <StyledCheckBoxWrapper onClick={OnTaskComplete} />
+      {isEditing ? (
+        <Input defaultValue={defaultValue} OnEditComplete={OnEditComplete} />

```

Comment:
atoms/inputでは小文字のo始まりです！
```suggestion
        <Input defaultValue={defaultValue} onEditComplete={OnEditComplete} />

```